README.md

```diff
@@ -49,9 +51,11 @@ If you're facing issues with setting up or using Chat With Your Data, follow the
 ### When should you use this repo?
 
 If you need to customize your scenario beyond what [Azure OpenAI on your data](https://learn.microsoft.com/azure/ai-services/openai/concepts/use-your-data) offers out-of-the-box, use this repository.
+    Note - developers can use this repo as starter and then they can customize the solution and adapt with best practices according to individual company policies. 
```

Comment:
I'm always nervous when I see "best practices" - I also don't know what this adds

Reply:
My suggestion is to add this sentence here, instead of the generic "best practices" recommendation:

By default, this repo comes with one specific set of RAG configurations including but not limited to: chunk size, overlap, retrieval/search type and system prompt. It is important that you evaluate the retrieval/search and the generation of the answers for your data and tune these configurations accordingly before you use this repo in production. For a starting point to understand and perform RAG evaluations, we encourage you to look into the [RAG Experiment Accelerator](https://github.com/microsoft/rag-experiment-accelerator).

Reply:
added the eval statement